Polygon_mesh_processing/include/CGAL/Polygon_mesh_processing/triangulate_hole.h

```diff
@@ -81,6 +83,24 @@ namespace Polygon_mesh_processing {
       \cgalParamExtra{If no valid triangulation can be found in this search space, the algorithm
                       falls back to the non-Delaunay triangulations search space to find a solution.}
     \cgalParamNEnd
+
+    \cgalParamNBegin{use_2d_constrained_delaunay_triangulation}
+      \cgalParamDescription{If `true`, use the 2D constrained Delaunay triangulation facet search space.}
```

Comment:
using CDT is not a method using a search space

Reply:
search space means that the algorithm is going to try all possible combinations of triangles from a set of possible triangles (= search space). DT3 is a restricted search space and the full search space is all possible triangles.

Reply:
You are right. I copy-pasted the text but did not notice that. I will fix that.